src/test/java/com/cloudant/tests/HttpProxyTest.java

```diff
@@ -50,12 +50,12 @@ public void proxyConfiguration() throws Exception {
         //create a client with a bogus address (TEST-NET)
         String mockProxyUser = "alpha";
         String mockProxyPass = "alphaPass";
-        CloudantClient client = new CloudantClient("http://192.0.2.0", "", "", new ConnectOptions()
+        CloudantClient client = new CloudantClient("http://192.0.2.0:80", new ConnectOptions()
```

Comment:
Good spot, that was working around a problem where default ports were not used (resulting in -1 for the port, that is now fixed so I can update).